### PR TITLE
fix: shooter tuning and updated paths

### PR DESCRIPTION
**Pull request by TiaSinghania**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 1]
```diff
@@ -162,6 +165,12 @@ public void periodic() {
 
 			SmartDashboard.putBoolean("is shooter primed", isShooterPrimed());
 		}
+
+		SmartDashboard.putNumber("Bottom Shooter RPM", Utils.toRPM(m_bottomFlywheel.getSelectedSensorVelocity()));
+		SmartDashboard.putNumber("Top Shooter RPM", Utils.toRPM(m_topFlywheel.getSelectedSensorVelocity()));
+		SmartDashboard.putBoolean("top at setpoint", m_topShooterPID.atSetpoint());
+		SmartDashboard.putBoolean("bottom at setpoint", m_bottomShooterPID.atSetpoint());
+
```
Comment (LennyAbbas) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:173]: why?

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 2]
```diff
@@ -109,7 +109,10 @@ public void periodic() {
 		}
 
 		if (m_feederTimer.get() > 0) {
-			m_topFeeder.set(ShooterConstants.kTopFeederSpeedFast);
+			if (m_bottomShooterPID.atSetpoint() && m_topShooterPID.atSetpoint()) {
+				m_topFeeder.set(ShooterConstants.kTopFeederSpeedFast);
+			}
+
```
Comment (LennyAbbas) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:115]: I assume this works but it is not implemented well. At this point if it works it's ok to merge.

**Review by LennyAbbas — CHANGES_REQUESTED** [review]
The pathweaver changes should have been done on a different branch but since it's already here and we're so close to competition, we can merge it in. Just make sure you update the description to say what changes are made

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java (hunk 2 above)
Comment (ReeledWarrior14) [reply src/main/java/frc/robot/subsystems/ShooterSubsystem.java:115]: It works, I can't really think of another way to implement it at the moment and we just need it working for competition.

**Review by ReeledWarrior14 — APPROVED** [review]

**Review by LennyAbbas — APPROVED** [review]

**Pull request merged**